src/SDKs/SqlManagement/Management.Sql/Generated/SdkInfo_SqlManagementClient.cs

```diff
@@ -94,10 +97,10 @@ internal static partial class SdkInfo
       // BEGIN: Code Generation Metadata Section
       public static readonly String AutoRestVersion = "latest";
       public static readonly String AutoRestBootStrapperVersion = "autorest@2.0.4283";
-      public static readonly String AutoRestCmdExecuted = "cmd.exe /c autorest.cmd https://github.com/Azure/azure-rest-api-specs/blob/master/specification/sql/resource-manager/readme.md --csharp --version=latest --reflect-api-versions --csharp-sdks-folder=E:\\Github2\\azure-sdk-for-net\\src\\SDKs";
```

Comment:
Please regenerate the code from spec merged in master

Reply:
done